### ISSUE TITLE
README: video introduction video and image no longer valid

### ISSUE THREAD

**Issue by ericherman** [standard-for-public-code/standard-for-public-code#1048]
The linked page https://www.youtube.com/watch?v=QWt6vB-cipE resolves with a page stating "This video isn't available anymore".

The linked image https://img.youtube.com/vi/QWt6vB-cipE/mqdefault.jpg gives a 404 error.

**Comment by Ainali** [comment]
Yes, the entire Foundation for Public Code YouTube is unavailable, and my guess it is the same thing happening when Boris's email got removed (but this time it was probably @ElenaFdR's). It could be resolved by someone with Google super admin rights assigning a new person to be the owner of the YouTube channel, but I am not sure if that would be @nebster or @amydarling (or either).

**Comment by ericherman** [comment]
Multiple solutions are possible, including:

 * remove the image and video
 * recover the video and images and host these via peertube
   * https://peertube.tv/search?search=publiccode
   * https://joinpeertube.org/publish-videos
 * replace the image and video with Wayback Machine links
    * https://web.archive.org/web/20230420161326/https://www.youtube.com/watch?v=QWt6vB-cipE
    * https://web.archive.org/web/20240302063149if_/https://img.youtube.com/vi/QWt6vB-cipE/mqdefault.jpg
 * recover the video and images and re-upload to youtube

**Comment by amydarling** [comment]
Hi,
I am happy to try and help.  I am not familiar with the youtube channel -
so if you send me a pointer to it, I can try to change the ownership.

Kind regards,
Amy

Amy Darling
cell: +1.781.354.0125

**Comment by Ainali** [comment]
@amydarling I think the explanation is on this help page: https://support.google.com/youtube/answer/4628007?hl=en&co=GENIE.Platform%3DDesktop#zippy=%2Cview-or-add-roles-to-your-brand-account 
(Do also check the expandable questions below the main content.)

**Comment by amydarling** [comment]
Hi Jan,
Unfortunately, I do not have access to any of the brand accounts for
YouTube.
@Ben Cerveny ***@***.***> can you please see if you have access and
can re-assign ownership?

Kind regards,
Amy

Amy Darling
cell: +1.781.354.0125

**Comment by nebster** [comment]
Hello Jan and Eric!  None of us have access to the Branded Account at the gmail workspace.  Apparently that needed to be explicitly assigned by you two, @ElenaFdR <https://github.com/ElenaFdR>, or Kehinde before departure.

I suppose we need to speak with Google about it.

thanks for the heads up!

Ben Cerveny, President
Foundation for Public Code <https://publiccode.net/>
***@***.*** | @neb

**Comment by ElenaFdR** [comment]
Sorry about this. [This also suggests contacting support.](https://stackoverflow.com/questions/65687894/no-access-to-the-owner-of-the-google-brand-account-anymore)

**Comment by ixuz** [comment]
The `publiccode.yml` still has the broken video link, but the rest seems to have been fixed.
See [publiccode.yml](https://github.com/standard-for-public-code/standard-for-public-code/blob/8efbc2b45ede6df2a779534718eefb9ba36463e9/publiccode.yml#L44)

**Comment by ericherman** [comment]
fixed 0fcf1668d0a0a34deaa5bff3337bb1ee0a45e21b
